packages/plugin-lighthouse/CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+## Setup
+
+Make sure to install dependencies:
+
+```sh
+npm install
+```
+
+### Chrome path
```

Comment:
I am wondering whether the Chrome path belongs to the `CONTRIBUTING.md` as opposed to the plugin's `README.md`.

If I try to run the plugin locally as a part of my Code PushUp setup, won't I encounter the same issue? 🤔

Reply:
Good question!

I consider it in the PR for the Plugin shell that include docs.